week2/2-thursday/homework/exercise9/RobPEx9.js

```diff
@@ -0,0 +1,14 @@
+//Mine has errors
+//Sort an Array 3
+//NOT DONE!!
+var arr3 = [
+    [1, 3, 4],
+    [2, 4, 6, 8],
+    [3, 6]
+  ];
+
+  var sum = arr3.sort(function(a) {
+      return a.reduce;
+  },0);
+
```

Comment:
On the right track here. Hint is remember the sort compare function takes two arguments: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort